cli/inputflags/flags.go

```diff
@@ -2,24 +2,29 @@ package inputflags
 
 import (
 	"encoding/json"
+	"errors"
 	"flag"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"regexp"
 
+	"github.com/brimsec/zq/cli/auto"
 	"github.com/brimsec/zq/zbuf"
 	"github.com/brimsec/zq/zio"
 	"github.com/brimsec/zq/zio/azngio"
 	"github.com/brimsec/zq/zio/detector"
 	"github.com/brimsec/zq/zio/ndjsonio"
+	"github.com/brimsec/zq/zio/zngio"
 	"github.com/brimsec/zq/zng/resolver"
 )
 
 type Flags struct {
 	zio.ReaderOpts
 	// The JSON type config is loaded from the types filie when Init is called.
 	jsonTypesFile string
+	ReadSize      auto.Bytes
+	ReadMax       auto.Bytes
```

Comment:
Nit: Keep exported fields at the top.
```suggestion
	zio.ReaderOpts
	ReadMax  auto.Bytes
	ReadSize auto.Bytes
	// The JSON type config is loaded from the types filie when Init is called.
	jsonTypesFile string
```